accessibility/tools.md

```diff
@@ -0,0 +1,93 @@
+# Accessibility tools
+
+- [Automated testing](#automated-testing)
+  - [Pa11y and axe](#pa11y-and-axe)
+- [Manual testing](#manual-testing)
+- [Assistive technology](#assistive-technology)
+  - [Screen readers](#screen-readers)
+- [Other accessibility tools](#other-accessibility-tools)
+  - [Accessibility testing tools in the browser](accessibility-testing-tools-in-the-browser)
+  - [Tools for manually analysing web pages](tools-for-manually-analysing-web-pages)
+- [Simulators](#simulators)
+
+## Automated testing
+
+Automated accessibility testing can be used to detect the most common accessibility problems. This is only the first step in the process; you MUST NOT rely on automated testing as your sole method of verification! 
+
+The GDS Accessibility team ran an experiment in 2017 to [audit the performance of a large number of available automated accessibility testing tools](https://accessibility.blog.gov.uk/2017/02/24/what-we-found-when-we-tested-tools-on-the-worlds-least-accessible-webpage/). They found that the highest number of errors that one tool was able to detect was 40%. The majority of the tools fell into the 20%-30% range. 
+
+
+### Pa11y and axe
+
+We use [Pa11y](http://pa11y.org/) to perform automated accessibility testing. Pa11y is an automated accessibility testing tool that supports [HTML_Codesniffer](https://squizlabs.github.io/HTML_CodeSniffer/) by Squizlabs, and [axe](https://www.deque.com/axe/) by Deque. 
+
+Projects MUST have Pa11y integrated at the build stage, and builds MUST fail if errors are introduced. You MUST NOT allow accessibility regressions to enter your project's codebase. 
+
```

Comment:
> You MUST NOT allow accessibility regressions to enter your project's codebase
Should we mention the reasoning here i.e. The risk of being in breach of contract with regard to our VPATs

Reply:
How about

```suggestion
Projects MUST have Pa11y integrated at the build stage, and builds MUST fail if errors are introduced. You MUST NOT allow accessibility regressions to enter your project's codebase. We're aiming to make accessibility better, so our baseline is to not make it worse. 
```

?

Reply:
yep, works for me